packages/build-tools/src/utils/expoUpdates.ts

```diff
@@ -88,14 +93,38 @@ export async function configureExpoUpdatesIfInstalledAsync(
   }
 
   const appConfigRuntimeVersion = ctx.job.version?.runtimeVersion ?? resolvedRuntimeVersion;
+
   if (ctx.metadata?.runtimeVersion && ctx.metadata.runtimeVersion !== appConfigRuntimeVersion) {
     ctx.markBuildPhaseHasWarnings();
-    ctx.logger.warn(
-      `Runtime version from the app config evaluated on your local machine (${ctx.metadata.runtimeVersion}) does not match the one resolved here (${appConfigRuntimeVersion}).`
-    );
-    ctx.logger.warn(
-      "If you're using conditional app configs, e.g. depending on an environment variable, make sure to set the variable in eas.json or configure it with EAS Secret."
-    );
+    ctx.logger.warn('Runtime version mismatch');
+    ctx.logger.warn(`Runtime version on your local machine: ${ctx.metadata.runtimeVersion}`);
+    ctx.logger.warn(`Runtime version calculated on EAS: ${appConfigRuntimeVersion}`);
+
+    if (
+      ctx.metadata?.projectFingerprintUrl &&
+      resolvedFingerprintSources &&
+      resolvedRuntimeVersion
+    ) {
+      try {
+        const result = await fetch(ctx.metadata?.projectFingerprintUrl);
```

Comment:
We will need to handle both `FingerprintSourceType.URL` and `FingerprintSourceType.PATH` here

Reply:
Could I handle the `FingerprintSourceType.PATH` in a separate PR?